gen2-seven-segment-recognition/download.py

```diff
@@ -0,0 +1,5 @@
+import gdown
+
+if __name__ == "__main__":
+
+    gdown.download("https://drive.google.com/uc?id=1LJG6xBnJvhHrVkNgsW1C7xaUGGD7totT", output="blobs/7_SEG_256x750_MONO_simplified_openvino_2021.4_6shave.blob")
```

Comment:
Could we rather add the model to depthai-model-zoo, and use blobconverter to download it?